index.html

```diff
@@ -33,9 +33,8 @@
 	<header class="showcase">
 	<div class="container showcase-inner">
 		<br><br><br><br><br><br><br><br><br><br>
-		<h1>Z U L U</h1>
-		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
-			Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut.</p>
+		<h1>z u l u</h1>
+		<p>It is important to take care of the patient, to be followed by the doctor, but it is a time of great pain and suffering. For let me come to the smallest detail, no one should 			   practice any kind of work except that.</p>

```

Comment:
placeholder text changed for ease of testing website (google would ask to translate from latin)